accounts-db/src/accounts_index/bucket_map_holder.rs

```diff
@@ -67,6 +69,12 @@ pub struct BucketMapHolder<T: IndexValue, U: DiskIndexValue + From<T> + Into<T>>
     _phantom: PhantomData<T>,
 
     pub(crate) startup_stats: Arc<StartupStats>,
+
+    /// Threshold entry count per bin for flushing
+    /// None for Minimal/InMemOnly, Some(entries_per_bin) for Threshold
+    /// Calculated as: 2^log2(entries_per_bin) * 7/8
+    /// where entries_per_bin accounts for 7/8 hashmap load factor in weighted bytes per entry.
+    threshold_entries_per_bin: Option<usize>,
```

Comment:
Can the name include 'flush', please?

Some suggestions:
* flush_threshold_per_bin
* flush_threshold
* flush_threshold_entries_per_bin